package.json

```diff
@@ -1,12 +1,10 @@
 {
   "name": "email-service-backend",
   "version": "1.0.0",
-  "private": true,
   "description": "A collection of frequently needed microservices",
-  "main": "index.js",
   "scripts": {
-    "start": "nodemon index.js",
-    "deploy:prod": "node index.js",
+    "start": "moleculer-runner --env --hot services/email/email.js services/api/api.js",
+    "deploy:prod": "moleculer-runner --env services/email/email.js ervices/api/api.js",
```

Comment:
@alias-rahil ~~ervices~~ services

Reply:
fixed in https://github.com/csivitu/email-service-backend/pull/3/commits/e8d893652e2960b1832402e5175f56f1f94fa79a